index.bs

```diff
@@ -90,60 +105,40 @@ dictionary CSSStyleSheetInit {
-			Set <var>sheet’s</var> origin-clean flag.
-		2. If the {{CSSStyleSheetInit/media}} attribute of <var>options</var> is a string,
-			<a>create a MediaList object</a> from the string
-			and assign it as <var>sheet’s</var> media.
-			Otherwise, assign a copy of the value of the attribute as <var>sheet’s</var> media.
-		3. If the {{CSSStyleSheetInit/alternate}} attribute of <var>options</var> is true,
-			set <var>sheet’s</var> alternate flag.
-		4. If the {{CSSStyleSheetInit/disabled}} attribute of <var>options</var> is true,
-			set <var>sheet’s</var> disabled flag.
-		5. Return <var>sheet</var>.
-	</dd>
 </dl>
 
 Using Constructed Stylesheets {#using-constructed-stylesheets}
 =============================
 
-A {{CSSStyleSheet}} can only be applied to the {{Document}} it is constructed on (e.g. the document on which the factory function is called on),
-and any {{ShadowRoot}} in the {{Document}} it is constructed on, by assigning an array containing the sheet to their {{adoptedStyleSheets}}.
-So, a stylesheet can be used in multiple {{DocumentOrShadowRoot}}s within the document it is constructed on.
-
-Non-explicitly constructed stylesheets cannot be added to {{adoptedStyleSheets}}.
-If {{adoptedStyleSheets}} got assigned an array that contains style sheets not made by {{createCSSStyleSheet(text)}}, {{createCSSStyleSheetSync(text)}} or {{createEmptyCSSStyleSheet}},
-a {{TypeError}} will be thrown.
-
 <pre class='idl'>
 partial interface DocumentOrShadowRoot {
 	attribute FrozenArray&lt;CSSStyleSheet> adoptedStyleSheets;
 };
 </pre>
 
+
 <dl>
 	<dt><dfn attribute for=DocumentOrShadowRoot lt="adoptedStyleSheets">adoptedStyleSheets</dfn></dt>
 	<dd>
-		Style sheets assigned to this attribute are part of the <a spec=cssom-1>document CSS style sheets</a>.
-		They are ordered after the stylesheets in {{Document/styleSheets}}.
+		On getting, {{adoptedStyleSheets}} returns this DocumentOrShadowRoot's [=adopted stylesheets=].
+
+		On setting, {{adoptedStyleSheets}} performs the following steps:
+
+		1. Let <var>adopted</var> be the result of converting the given value to a FrozenArray&lt;CSSStyleSheet>
+		2. If any entry of <var>adopted</var> has no associated document (e.g. it's not made by factory methods to construct stylesheets), throw a {{NotAllowedError}} {{DOMException}}. 
+		3. Set this {{DocumentOrShadowRoot}}'s adopted stylesheets to <var>adopted</var>.
 	</dd>
 </dl>
 
-Applying Styles In All Contexts {#styles-in-all-contexts}
-===================
+Every {{DocumentOrShadowRoot}} has <dfn>adopted stylesheets</dfn>.
+
+The user agent must include all style sheets in the {{DocumentOrShadowRoot}}'s
+[=adopted stylesheets=] inside its <a spec=cssom-1>document CSS style sheets</a> collection if it is a {{Document}}, or shadow CSS style sheets collection if it is a {{ShadowRoot}}. The [=adopted stylesheets=] are ordered after all the other style sheets (i.e. those derived from {{DocumentOrShadowRoot/styleSheets}}).
```

Comment:
Hey, I figured out the solution. Just use `<a>document or shadow root CSS style sheets</a>` and it will link to https://drafts.csswg.org/cssom/#documentorshadowroot-document-or-shadow-root-css-style-sheets, which does what you want.

Reply:
oh it was right there! Thanks.